Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6723,7 +6723,7 @@ UseGPUProcessForMediaEnabled:
   humanReadableName: "GPU Process: Media"
   humanReadableDescription: "Do all media loading and playback in the GPU Process"
   webcoreBinding: none
-  condition: ENABLE(GPU_PROCESS) && !USE(GSTREAMER)
+  condition: ENABLE(GPU_PROCESS) && !USE(GSTREAMER) && !PLATFORM(COCOA)
```

Comment:
I don't understand this change. Don't we do media playback in the GPU Process on Cocoa? I know loading is handled in WebContent, but I'm surprised to see this flag be turned off.

Reply:
That is a good point. The intention behind this change was not to change any behavior, but to make it impossible to toggle this setting off on Cocoa platforms. But this is perhaps not the right way to do that?

Thanks for reviewing!